Packs/GitHubITAdmin/Integrations/GitHub_IT_Admin/GitHub_IT_Admin.py

```diff
@@ -0,0 +1,615 @@
+
+        else:
+            user_term = "\"" + user_term + "\""
+            uri = f'scim/v2/organizations/{encode_string_results(self.org)}/' \
+                  f'Users?filter={encode_string_results(input_type)} eq {encode_string_results(user_term)}'
+        return self.http_request(
+            method='GET',
+            url_suffix=uri,
+        )
+
+    def create_user(self, data):
+        uri = f'scim/v2/organizations/{encode_string_results(self.org)}/Users'
+        return self.http_request(
+            method='POST',
+            url_suffix=uri,
+            data=data,
+        )
+
+    def update_user(self, user_term, data):
+        uri = f'scim/v2/organizations/{encode_string_results(self.org)}/Users/{encode_string_results(user_term)}'
+        return self.http_request(
+            method='PUT',
+            url_suffix=uri,
+            data=data,
+        )
+
+    def disable_user(self, data):
+
+        uri = f'scim/v2/organizations/{encode_string_results(self.org)}/Users/{encode_string_results(data)}'
+        return self.http_request(
+            method='DELETE',
+            url_suffix=uri
+        )
+
+    # Builds a new user github_user dict with pre-defined keys and custom mapping (for user)
+    def build_github_create_user(self, args, scim):
+        extension_schema = scim.get(SCIM_EXTENSION_SCHEMA)
+
+        github_user = scim
+
+        custom_mapping = None
+        if args.get('customMapping'):
+            custom_mapping = json.loads(args.get('customMapping'))
+        elif CUSTOM_MAPPING_CREATE:
+            custom_mapping = json.loads(CUSTOM_MAPPING_CREATE)
+
+        if custom_mapping and extension_schema:
+            for key, value in custom_mapping.items():
+                # key is the attribute name in input scim. value is the attribute name of app profile
+                user_extension_data = extension_schema.get(key)
+                if user_extension_data:
+                    github_user[value] = user_extension_data
+
+        return github_user
+
+    # Builds a new user github profile dict with pre-defined keys and custom mapping (for user)
+    def build_github_update_user(self, args, github_user, scim):
+        extension_schema = scim.get(SCIM_EXTENSION_SCHEMA)
+        custom_mapping = None
+        if args.get('customMapping'):
+            custom_mapping = json.loads(args.get('customMapping'))
+        elif CUSTOM_MAPPING_UPDATE:
+            custom_mapping = json.loads(CUSTOM_MAPPING_UPDATE)
+
+        if custom_mapping and extension_schema:
+            for key, value in custom_mapping.items():
+                # key is the attribute name in input scim. value is the attribute name of app profile
+                user_extension_data = extension_schema.get(key)
+                if user_extension_data:
+                    github_user[value] = user_extension_data
+
+        return github_user
+
+
+class OutputContext:
```

Comment:
I think it would be easier for you to use our built in CommandResults - see here for further information: https://xsoar.pan.dev/docs/integrations/code-conventions#commandresults

Reply:
Again it is a part of design and OutputContext is being used in all the integrations for User provisioning.